spock-core/src/main/java/org/spockframework/mock/runtime/GroovyMockFactory.java

```diff
@@ -14,17 +14,24 @@
 
 package org.spockframework.mock.runtime;
 
-import org.spockframework.mock.*;
-import org.spockframework.runtime.GroovyRuntimeUtil;
-import spock.lang.Specification;
-
 import java.lang.reflect.Modifier;
 import java.util.ArrayList;
+import java.util.List;
 
-import groovy.lang.*;
+import org.spockframework.mock.CannotCreateMockException;
+import org.spockframework.mock.IMockConfiguration;
+import org.spockframework.mock.IMockFactory;
+import org.spockframework.mock.MockImplementation;
+import org.spockframework.mock.MockNature;
+import org.spockframework.runtime.GroovyRuntimeUtil;
+import org.spockframework.util.ReflectionUtil;
```

Comment:
+1 for not using *-import

Reply:
actually that is something that I still need to fix, the new "official" style will prefer *-imports if it is more than 3 IIRC

Reply:
I tended to like *-imports in the past, but over time I realized that it can hide complexity of a given class, regarding number of class level dependencies. There could be 5 star imports which under the hood import 40+ classes. While in some technical/utility/controller classes it can be fine/sane/required, usually it is good to be aware of that (especially that Idea hides imports by default, so it's doesn't reduce readability or effectiveness of working with that files). That's my two cents.